### ISSUE TITLE
Defaults for a job creation: `worker queue` (`all.q`) clashes with `wallclock timeout` (`3600`)

### ISSUE THREAD

**Issue by vuphan314** [StarExec/StarExec#315]
Default values (at least in this [space](https://www.starexec.org/starexec/secure/explore/spaces.jsp?id=521696)):
- worker queue: `all.q`
- wallclock timeout: `3600`

But the wallclock timeout `3600` appears to require the worker queue `long.q`.

I suggest changing a default value:
- worker queue to `long`, or 
- wallclock timeout to `1800`.

Screenshot of a job creation:
![Screenshot from 2022-05-21 09-59-06](https://user-images.githubusercontent.com/12940605/169657255-e187db85-9905-4cba-9789-a243e419c8f3.png)

Thank you.

**Comment by jwaldmann** [comment]
These defaults (wallclock) are set by the community leader(s)? E.g., for Termination, it's different numbers.

**Comment by astump** [comment]
Yes, please contact the community leader on StarExec to request a change.
Let me know if there are any difficulties.
